### ISSUE TITLE
Accessibility: adding users to lists

### ISSUE THREAD

**Issue by clarcharr** [tootsuite/mastodon#6152]
Right now, when adding users to a list, you have to click the (+) icon, which is very small. You should be able to click the entire row to add/remove a user.

Additionally, the (+) and (x) icons to indicate whether the user is currently added or removed isn't very easy to spot and makes it a bit difficult to tell whether you've added someone or not. There should be a colour change and some other indicator to show whether a user was added or not.

**Comment by andi1984** [comment]
@clarcharr Yes, I also think so that we can improve A11Y there. Also the "x" icon of the outer modal is very confusing – at least for me (see screenshot).

<img width="389" alt="bildschirmfoto 2018-01-10 um 22 34 48" src="https://user-images.githubusercontent.com/1077917/34796701-9cd7e448-f656-11e7-84ec-9e29208ed91c.png">

**Comment by ciphercharly** [comment]
also an add-to-list button when on followed account home menu (see pic) would be very useful!

![screenshot from 2018-04-10 08-08-31](https://user-images.githubusercontent.com/9661894/38539014-96ef1bec-3c96-11e8-9208-5358894e4bc8.png)

**Comment by manuelviens** [comment]
Came there specifically to report all of theses when I found this already opened issue.
It's a pain actually to add users to lists. And yes, it's not clear the gray "+" and "x" icons.
:thumbup: to all previous comments.

**Comment by luzpaz** [comment]
Should be labeled 'accessibility'
